### PR TITLE
Add websocket example with serverless

### PR DESCRIPTION
**Pull request by gjgd**
https://github.com/LesGars/WatchWithMe/projects/1#card-36544760

### CODE REVIEW

File: backend/handler.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const AWS = require("aws-sdk");
+
+module.exports.connect = async (event, context) => {
+    return {
+        statusCode: 200,
+    };
+};
+
+module.exports.disconnect = async (event, context) => {
```
Comment (MollardMichael) [inline backend/handler.js:9]: This handler is unused in your current configuration no ?

I think i'd be better to link it to the $disconnect event of the API gateway

File: backend/package.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (MollardMichael) [inline backend/package.json:1]: Pouruqi un package.json. C'est utile dans le futur ?

File: backend/handler.js (hunk 1 above)
Comment (gjgd) [reply backend/handler.js:9]: Yes mb, will do that

File: backend/package.json (hunk 2 above)
Comment (gjgd) [reply backend/package.json:1]: It'll be useful when we want to use dependencies other than `aws-sdk` in the backend (which is automatically included in the context of all AWS lambdas).

But yeah technically it's not needed now. Removing it

File: Makefile [hunk 3]
```diff
@@ -1,8 +1,9 @@
 help: ## List available commands
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-install-pnpm: ## Install/Update pnpm package manager
+install-deps: ## Install/Update globale packages
```
Comment (Startouf) [inline Makefile:4]:
```suggestion
install-deps: ## Install/Update global packages
```

**Pull request merged**